x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -189,7 +189,7 @@ export const getColumns = ({
       align: 'center',
       render: (tags: Rule['tags']) => {
         if (tags.length === 0) {
-          return getEmptyTagValue();
+          return null;
```

Comment:
Nit and probably out of the scope of this PR:

The name `getEmptyTagValue` looks weird when used in other columns:

```tsx
      render: (value: Rule['status_date']) => {
        return value == null ? (
          getEmptyTagValue()
        ) : (
          <FormattedRelativePreferenceDate
            tooltipFieldName={i18n.COLUMN_LAST_COMPLETE_RUN}
            relativeThresholdInHrs={DEFAULT_RELATIVE_DATE_THRESHOLD}
            value={value}
            tooltipAnchorClassName="eui-textTruncate"
          />
        );
      },
```

Probably we could rename it to `getEmptyCellValue()`.

Reply:
Absolutely 👍

Reply:
@banderror , FYI: https://github.com/elastic/kibana/issues/119756